src/AudioManagerImpl.js

```diff
@@ -233,6 +233,16 @@ export default class AudioManagerImpl implements AudioManager {
         }
     }
 
+    playStringMessage(message: string) {
```

Comment:
This is very close in function to `playAnnouncement`, with quite a bit of duplicated code.  I would suggest refactoring `playAnnouncement` to just look up the internationalised content and have it use `playStringMessage` to check `announcementsEnabled` and play the actual announcement if needed.

Reply:
It might also be good to make `playAnnouncement` something like `playMessageId` to more closely match its new function.

Reply:
Thanks for the comment @the-t-in-rtf, that makes sense to refactor playAnnouncement and use playStringMessage to play the announcement. About the other suggestions, we would need two separate functions for the announcements, one for preview and one for feedback, so I am thinking changing the name to playFeedbackAnnouncement and have the preview one named playPreviewAnnouncement would make more sense.